cyences_app_for_splunk/default/data/ui/views/cs_overview.xml

```diff
@@ -238,30 +238,40 @@
 | rename "Notable Events" as Count 
 | append 
     [| search `cs_sophos` type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") dhost IN ("*") user IN ("*") earliest=$timeRange.earliest$ latest=$timeRange.latest$
     | fields _time host dhost type 
     | stats latest(type) as type by host, dhost 
     | where type=="Event::Endpoint::ServiceNotRunning" 
-    | stats count as Count
+    | stats count as Count | search [| search `cs_sophos` earliest=$timeRange.earliest$ latest=$timeRange.latest$ | head 1 | stats count | eval search=if(count&gt;0, "", "DONOTRETURNANYRESULTS") | fields search | format]
     | eval Name="Currently Disabled/Stopped Sophos Services"] 
 | append
     [| search `cs_sophos` type="Event::Endpoint::Sav*" host="*" dhost IN ("*") user IN ("*") earliest=$timeRange.earliest$ latest=$timeRange.latest$
     | fields host dhost type 
     | stats latest(type) as type by host, dhost 
     | where type=="Event::Endpoint::SavDisabled" 
-    | stats count as Count 
+    | stats count as Count | search [| search `cs_sophos` earliest=$timeRange.earliest$ latest=$timeRange.latest$ | head 1 | stats count | eval search=if(count&gt;0, "", "DONOTRETURNANYRESULTS") | fields search | format]
     | eval Name="Currently Stopped/Disabled Sophos RealTime Protection"]
 | append
-    [| search ((`cs_wineventlog_system` SourceName="Microsoft-Windows-Service Control Manager" Message="The Windows Defender Antivirus Service service entered the *" Message=*stopped*) OR (`cs_windows_defender` EventCode IN (1150, 1151))) earliest=$timeRange.earliest$ latest=$timeRange.latest$ | fields EventCode, host | stats latest(EventCode) as EventCode by host | stats count(eval(EventCode!=1150 AND EventCode!=1151)) as Count | eval Name="Currently Disabled/Stopped Windows Defender Services"]
+    [| search ((`cs_wineventlog_system` SourceName="Microsoft-Windows-Service Control Manager" Message="The Windows Defender Antivirus Service service entered the *" Message=*stopped*) OR (`cs_windows_defender` EventCode IN (1150, 1151))) earliest=$timeRange.earliest$ latest=$timeRange.latest$ | fields EventCode, host | stats latest(EventCode) as EventCode by host | stats count(eval(EventCode!=1150 AND EventCode!=1151)) as Count | search [| search `cs_windows_defender` earliest=$timeRange.earliest$ latest=$timeRange.latest$ | head 1 | stats count | eval search=if(count&gt;0, "", "DONOTRETURNANYRESULTS") | fields search | format] | eval Name="Currently Disabled/Stopped Windows Defender Services"]
```

Comment:
| search ((`cs_wineventlog_system` SourceName="Microsoft-Windows-Service Control Manager" Message="The Windows Defender Antivirus Service service entered the *" Message=*stopped*) OR (`cs_windows_defender` EventCode IN (1150, 1151))) earliest=$timeRange.earliest$ latest=$timeRange.latest$  [| search `cs_windows_defender` earliest=$timeRange.earliest$ latest=$timeRange.latest$ | head 1 | stats count | eval search=if(count&gt;0, "", "sourcetype=DONOTRETURNANYRESULTS") | fields search | return $search]| fields EventCode, host | stats latest(EventCode) as EventCode by host | stats count(eval(EventCode!=1150 AND EventCode!=1151)) as Count   | eval Name="Currently Disabled/Stopped Windows Defender Services"

Reply:
Bro Let's use something like above. More optimised query.

Reply:
Done.

Reply:
Bro, can we do these changes everyplace where we are using DONOTRETURNANYRESULTS on this dashboard.

Reply:
Ohh, I see. Done